collectors/COLLECTORS.md

```diff
@@ -225,7 +231,13 @@ configure any of these collectors according to your setup and infrastructure.
 -   [Pi-hole](https://learn.netdata.cloud/docs/agent/collectors/go.d.plugin/modules/pihole/): Monitor basic (DNS
     queries, clients, blocklist) and extended (top clients, top permitted, and blocked domains) statistics using the PHP
     API.
--   [PowerDNS](/collectors/python.d.plugin/powerdns/README.md): Monitor authoritative server and recursor statistics.
+-   [PowerDNS Authoritative Server
+    (Go)](https://learn.netdata.cloud/docs/agent/collectors/go.d.plugin/modules/powerdns): Monitor one or more instances
```

Comment:
should be one line?

Reply:
It looks fine in the generated HTML. My editor is separating them onto multiple lines because I have a line limit of 120 characters, and it needs to wrap that long URL.

Reply:
ok, i see, it was not the only place with strange (imo) line separation. Yeah, it looks fine!